src/robot/variables/search.py

```diff
@@ -83,7 +83,7 @@ def __nonzero__(self):
     def __unicode__(self):
         if not self:
             return '<no match>'
-        items = ''.join('[%s]' % i for i in self.item) if self.items else ''
+        items = ''.join('[%s]' % i for i in self.items) if self.items else ''
```

Comment:
Good catch, there's clearly a bug in the existing code! Luckily this is new code in RF 3.2.